sound/soc/intel/boards/sof_board_helpers.c

```diff
@@ -248,6 +248,48 @@ int sof_intel_board_set_intel_hdmi_link(struct device *dev,
 }
 EXPORT_SYMBOL_NS(sof_intel_board_set_intel_hdmi_link, SND_SOC_INTEL_SOF_BOARD_HELPERS);
 
+int sof_intel_board_set_amp_link(struct device *dev,
```

Comment:
It is for SSP amp only, better to rename to ` sof_intel_board_set_ssp_amp_link`. We may have a similar helper for sdw amp someday.

Reply:
done